### DISCUSSION TITLE
Bryan Jenks, What is your theme.

### DISCUSSION THREAD

**Discussion by DaniyalAhmed-kh** [BryanJenksCommunity/FAQ#51] [Q&A] [answered]
### Question

What is your question?

Bryan  Jenks! I know you are very busy and have alot going on but it will be really helpful if you answer. Please briefly tell me what theme are you using and what are the snippets you have added to your Obsidian Vault. I saw your videos and your published vault and it's beautiful. I also saw your zotero video and it had some snippet which made the links small in preview. Overall, Can you tell me what are the snippets and what theme you use for your vault. Thanks, 
Regards, Daniyal Ahmed.
---

By filing a question to this repo, I promise that

-   [x] I have searched open/closed issues on this repo for my answer already

I understand that my issue may be closed if I don't fulfill my promises.

**Comment by tallguyjenks** [comment, marked as answer]
All of the visual things you see in my vault are a result of my custom CSS, which i give away along with my templates in exchange for signing up to my email newsletter which you can do [[[HERE]]](https://bryanjenkstech.ck.page/d4ec0713d5)

:)

**Reply by itizarsa** [reply]
@tallguyjenks After subscription, I received a mail to confirm email. After clicking confirm, am able to download your vault as zip file, but the theme is different than the one you are using. Could help here?

**Reply by tallguyjenks** [reply]
@itizarsa my theme changes over time, the theme is the downloadable vault is the most recent one i have available, i haven't gotten around to updating things yet

**Reply by toalg** [reply]
It takes me to your website, where do i get the .css file from? Sorry for hyjacking your thread, im net to Obsidian and Coding in general. Any support is welcomed.

ps. Your youtube videos are great.

**Reply by tallguyjenks** [reply]
@toalg at what point does it take you to my site? if you signed up to the newsletter then the next step is opting in to the verification email sent to you, that will also take you to my site where a link for the download is kept

**Comment by toalg** [comment]
I found how to download it, I love your videos. However I do feel that
there is a certain skill level needed to grasp your videos. I am trying to
get to grips with Obsidian and it's a bit of a learning curve, Thanks for
the hours of content I have a great deal to work..
